src/transporters.ts

```diff
@@ -50,13 +50,12 @@ export class DefaultTransporter {
   configure(opts: AxiosRequestConfig = {}): AxiosRequestConfig {
     // set transporter user agent
     opts.headers = opts.headers || {};
-    if (!opts.headers['User-Agent']) {
+    const uaValue: string = opts.headers['User-Agent'];
+    if (!uaValue) {
       opts.headers['User-Agent'] = DefaultTransporter.USER_AGENT;
-    } else if (
-        opts.headers['User-Agent'].indexOf(DefaultTransporter.USER_AGENT) ===
-        -1) {
+    } else if (!uaValue.includes('google-api-nodejs-client/')) {
```

Comment:
Can you define a constant for `'google-api-nodejs-client/'` and use it here and also in the `DefaultTransporter.USER_AGENT` definition?

Reply:
👍